packages/react/src/components/Forms/Examples/index.stories.tsx

```diff
@@ -184,3 +195,114 @@ export const Sports = () => {
     </CunninghamProvider>
   );
 };
+export const SportsDisabled = () => {
+  const [withValues, setWithValues] = useState(false);
+  return (
+    <CunninghamProvider>
+      <form
+        style={{
+          display: "flex",
+          flexDirection: "column",
+          gap: "1rem",
+          width: "400px",
+        }}
+      >
+        <h1
+          className="fs-h3 fw-bold clr-greyscale-900"
+          style={{ textAlign: "center" }}
+        >
+          Register
+        </h1>
+        <div style={{ display: "flex", gap: "1rem" }}>
+          <Input
+            label="First name"
+            disabled={true}
+            value={withValues ? "John" : undefined}
+          />
+          <Input label="Last name" disabled={true} />
+        </div>
+
+        <div style={{ display: "flex", gap: "0.5rem" }}>
+          <Radio name="gender" label="Male" fullWidth={true} disabled={true} />
+          <Radio name="gender" label="Female" disabled={true} />
+          <Radio name="gender" label="Other" disabled={true} />
+        </div>
+        <DatePicker
+          label="Date of birth"
+          fullWidth={true}
+          disabled={true}
+          value={withValues ? "2023-05-23T00:00:00.000+00:00" : undefined}
+        />
+        <Select
+          disabled={true}
+          label="Competition"
+          options={[
+            {
+              label: "Athletics",
+            },
+            {
+              label: "Swimming",
+            },
+            {
+              label: "Marathon",
+            },
+          ]}
+          fullWidth={true}
+          value={withValues ? "Swimming" : undefined}
+        />
+        <Select
+          disabled={true}
+          label="Previous rewards"
+          multi={true}
+          options={[
+            {
+              label: "Bronze",
+            },
+            {
+              label: "Silver",
+            },
+            {
+              label: "Gold",
+            },
+            {
+              label: "Flocon",
+            },
+            {
+              label: "Ourson",
+            },
+            {
+              label: "Chamois",
+            },
+          ]}
+          fullWidth={true}
+          value={withValues ? ["Silver"] : undefined}
+        />
+        <DateRangePicker
+          startLabel="Start date"
+          endLabel="End date"
+          value={
+            withValues
+              ? [
+                  "2023-05-23T00:00:00.000+00:00",
+                  "2023-06-23T00:00:00.000+00:00",
+                ]
+              : undefined
+          }
+          fullWidth={true}
+          disabled={true}
+        />
+        <Switch
+          label="Enable values"
+          checked={withValues}
+          onChange={(e) => setWithValues(e.target.checked)}
+        />
```

Comment:
When we enable/disable values here, we can see that the select multi is changing size.
Select multi empty height should be the same as when it get one element in it.

Reply:
Yes I agree it was on purpose. But there is a opened discussion about this ... TBC